docs/feature-gates.md

```diff
@@ -138,3 +139,28 @@ foo           bar                   1          12        1221    2020-09-07T13:2
 #### Requirements for this Feature
 
 None
+
+### NodePortLocal
+
+`NodePortLocal` is a feature that runs as part of the Antrea Agent, through which
+each port of a Pod can be reached from external network using a port in the Node
+in which the Pod is running. In addition, to enabling NodePortLocal in features gates,
```

Comment:
Nit: in which -> on which

Reply:
Do you mean: "In addtion to enabling..."?

Probably: enabling the NodePortLocal feature gate